### DISCUSSION TITLE
permission denied while trying to connect to the Docker daemon socket

### DISCUSSION THREAD

**Discussion by na1983** [docker-library/postgres#1167] [General]
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/images/create?fromImage=postgres&tag=latest": dial unix /var/run/docker.sock: connect: permission denied

**Comment by LaurentGoderre** [comment]
This doesn't seem to be a problem with the image but with the way Docker itself is being called. Can you provide the command you used to get this error?
